src/routes/recycle/CreateNewRecycleOrder.jsx

```diff
@@ -0,0 +1,188 @@
+    0: "0 lbs",
+    200: { style: { whiteSpace: "nowrap" }, label: <span>200+ lbs</span> },
+  };
+
+  const navigate = useNavigate();
+
+  const handleReturnToRecyclePage = () => {
+    navigate("/recycle");
+  };
+
+  const handleBackButtonOnClick = () => {
+    navigate("/recycle");
+  };
+  return (
+    <>
+      {createOrder === 0 ? (
+        <div>
+          <HeaderNav
+            header="Create Recycle Order"
+            showBackButton={true}
+            showLogOutButton={true}
+            backButtonOnClick={handleBackButtonOnClick}
+          />
+          <Paragraph className="mx-6">
+            Use this page to input more detailed information and create a
+            recycle order. One of the recycle companies close to you will accept
+            your order and recycle for you.
+          </Paragraph>
+
+          <Flex vertical className="mx-8">
+            <Heading level={2}>Estimate the weight in Lbs</Heading>
+            <div className="mx-12 w-10/12">
+              <CustomInput
+                placeholder="0"
+                allowClear
+                value={inputValue}
+                onChange={handleInputChange}
+              />
+              <Slider
+                min={0}
+                max={200}
+                marks={marks}
+                value={inputValue}
+                onChange={handleSliderChange}
+                className="font-paragraph"
+              />
+            </div>
+          </Flex>
+          <Flex vertical className="my-6 mx-8">
+            <Heading level={2}>Classify the leftovers</Heading>
+            <div className="mx-12 w-10/12">
+              <CustomCheckbox
+                content="Intact Solid"
+                checked={checkedStates.intactSolid}
+                onChange={handleCheckboxChange("intactSolid")}
+              />
+              <CustomCheckbox
+                content="Partial Solid"
+                checked={checkedStates.partialSolid}
+                onChange={handleCheckboxChange("partialSolid")}
+              />
+              <CustomCheckbox
+                content="Liquid"
+                checked={checkedStates.liquid}
+                onChange={handleCheckboxChange("liquid")}
+              />
+            </div>
+          </Flex>
+
+          <Flex vertical className="my-6 mx-8">
+            <Heading level={2}>Pickup Time Today</Heading>
+            <div className="mx-12 w-10/12">
+              <Select
+                className="font-paragraph w-full h-10"
+                placeholder="Select one time slot"
+                onChange={handleTimeslotChange}
+                options={[
+                  { value: "20:00 - 21:00", label: "20:00 - 21:00" },
+                  { value: "21:00 - 22:00", label: "21:00 - 22:00" },
+                  { value: "22:00 - 23:00", label: "22:00 - 23:00" },
+                ]}
+              />
+            </div>
+          </Flex>
+
+          <Flex vertical center className="mx-20 my-12">
```

Comment:
Use `justify="center"` instead of `center` here if you want to center your contents in the `Flex` box, or you'll get warnings in console like in #15.

Reply:
This is kind of important because users are most likely to notice this since they can see the console during heuristic evaluation.